src/app/components/ms-contents/ms-contents.component.html

```diff
@@ -0,0 +1,52 @@
+<div class="msContents">
+    <evt-ms-desc-section [label]="'msContents'"></evt-ms-desc-section>
+    <div>
+        <ng-container *ngIf="data?.structuredData; else elseBlock">
+            <div class="section">
+                <div class="row no-gutters" *ngIf="data?.summary">
```

Comment:
If you want to use bootstrap, use it correctly. Check the grid layout rules at https://getbootstrap.com/docs/4.1/layout/overview/ 

N.B. In our case it's better to use "container-fluid". Check bootstrap guidelines

Reply:
In this case it's not necessary to use container fluid, because this component will be put in a panel which already has his own paddings. So remove "container-fluid" class. I'm sorry, my mistake!